underscore.js

```diff
@@ -90,6 +90,7 @@
   var cb = function(value, context, argCount) {
     if (value == null) return _.identity;
     if (_.isFunction(value)) return optimizeCb(value, context, argCount);
+    if (_.isRegExp(value)) return function(obj) { return !!value.exec(obj) };
```

Comment:
@olleicua I believe he's asking if we could use this method instead of `.exec()`: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/RegExp/test